lib/chromedriver.js

```diff
@@ -16,11 +16,11 @@ const DEFAULT_HOST = '127.0.0.1';
 const DEFAULT_PORT = 9515;
 class Chromedriver extends events.EventEmitter {
   constructor (args = {}) {
-    const {host, port, executable, cmdArgs, adbPort, verbose, logPath} = args;
+    const {host, port, executable, cmdArgs, adb, verbose, logPath} = args;
     super();
     this.proxyHost = host || DEFAULT_HOST;
     this.proxyPort = port || DEFAULT_PORT;
-    this.adbPort = adbPort;
+    this.adb = adb;
```

Comment:
or, if we don't want a breaking change, we could play nice, and allow both `adb` and `adbPort`. if the former is included we ignore the latter. if the latter is included but not the former, then we create a new `ADB` instance with the specified port, and assign it to `this.adb`.